### DISCUSSION TITLE
Show clickable code lines

### DISCUSSION THREAD

**Discussion by useross** [facelessuser/pyspelling#185] [General]
Hi, 
thanks for this nice tool.
Is there a way to have clickable links on which line the spelling mistake is?

**Comment by facelessuser** [comment]
Some context is lost in the translation of certain filters. It may be possible to at least add clickable links to files (depending on whether the terminal supports it or not). It is something to consider.  You can create an issue so I can remember to look into it as I have time.

**Reply by useross** [reply]
okay, thanks.
I guess the more important thing would be the link to the line.

**Reply by facelessuser** [reply]
You likely won't get the line. Some of the transformations are significant enough that the context is lost. Some parsing to HTML completely transforms the content. I can't universally guarantee line numbers. So if that is the hope, it unfortunately will not be the case.

**Reply by useross** [reply]
ok, thanks for the explanation :)
I will go on working without lines, this is also okay. Just wanted to inform me :)
